address/address.go

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2018 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package address
+
+import (
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/iotexproject/iotex-core/address/bech32"
+	"github.com/iotexproject/iotex-core/iotxaddress"
+	"github.com/iotexproject/iotex-core/logger"
+	"github.com/iotexproject/iotex-core/pkg/enc"
+	"github.com/iotexproject/iotex-core/pkg/hash"
+)
+
+// init reads IOTEX_NETWORK_TYPE environment variable. If it exists and the value is equal to "testnet" with case
+// ignored,  the global variable isTestNet is set to true for the whole runtime
+func init() {
+	isTestNet = strings.EqualFold(os.Getenv("IOTEX_NETWORK_TYPE"), "testnet")
+}
+
+// AddressLength indicates the byte length of an address
+const AddressLength = 25
+
+var (
+	isTestNet     bool
+	mainnetPrefix = "io"
+	testnetPrefix = "it"
+
+	// ErrInvalidAddr indicates the invalid address error
+	ErrInvalidAddr = errors.New("Invalid address")
+)
+
+// Address is the new address format to be used on IoTeX blockchain and subchains. It is composed of parts in the
+// following order:
+// 1. uint32: chain ID
+// 2. uint8: version of address
+// 3. 20 bytes: hash derived from the the public key
+type Address struct {
+	chainID uint32
+	version uint8
+	pkHash  [hash.PKHashSize]byte
```

Comment:
can use hash.AddrHash instead of [hash.PKHashSize]byte

Reply:
Fixed